### ISSUE TITLE
consider cursor position instead of selection when scrolling automatically

### ISSUE THREAD

**Issue by Ingegneus** [xournalpp/xournalpp#4476]
**Is your feature request related to a problem? Please describe.**
when you have a large selection (a screenshot or just a selection of strokes) that is similar in size or larger than your visible draw area and try to move/reposition your selection, the page automatically scolls up or down because the selection ist close to the edge of the draw area. this makes it next to impossible to move large selections around precisely, because you have to scroll out in order to not trigger the scrolling.
![image](https://user-images.githubusercontent.com/78076361/205609061-01f2f78f-a60d-4975-99f2-7fca6f37dea8.png)

**Describe the solution you'd like**
IMO it would be better to trigger the scrolling if the cursor is next to the edge and not the selection itself.

**Comment by K4LCIFER** [comment]
I have to agree that this behaviour has been troublesome; however, it could be argued that if the image is larger than the viewed area, you need to scroll out to position it properly anyways.
